client/src/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Thoughts of the day challenge">
```

Comment:
@alexander-daniel If I wanted to make the description's content dynamic based on an ENV var, how could I achieve that?

Reply:
You would use something like a webpack ENV plugin and pass it in as an ENV variable when you run `yarn build` or `webpack`. That would be injected into your code, and in conjunction with a proper HTML loader for webpack you could achieve this! 

You could also leverage a library like `react-helmet` to do this using JS. But you might miss out on SEO benefits if your `<meta>` tags are generated dynamically. Although search engine bots have improved a lot in the last while.